spark.js.org

lukemccrea26 opened the pull request:
- [x] There is reasonable content on the page (see: [Content](https://github.com/HammerheadStudios/SparkCode))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (commented):
I am slightly confused as to why you're requesting this on your fork, rather than the [upstream repo](https://github.com/HammerheadStudios/SparkCode) which you also appear to have push access to?

lukemccrea26 commented:
> I am slightly confused as to why you're requesting this on your fork, rather than the [upstream repo](https://github.com/HammerheadStudios/SparkCode) which you also appear to have push access to?

Sorry. I can delete this pull request and make another.

lukemccrea26 commented:
> I am slightly confused as to why you're requesting this on your fork, rather than the [upstream repo](https://github.com/HammerheadStudios/SparkCode) which you also appear to have push access to?

I was originally going to do this off the upstream one. I added a CNAME file to it and then it started redirecting to your website. I deleted it because you needed to be able to see the page content to approve me and you could not see the page if it redirected to your website. Even after I deleted the file it kept redirecting so I forked it and made the request off that so you could view the page.

MattIPv4 commented:
That redirect is expected, you can add the CNAME and we'll be able to check site content from GitHub itself, don't worry :)

lukemccrea26 commented:
I updated it.

lukemccrea26 commented:
> That redirect is expected, you can add the CNAME to the upstream repo and we'll be able to check site content from GitHub itself, don't worry :)
> 
> You can update this PR to target the original upstream repo 👍

I updated it.

MattIPv4 commented:
@indus What do you think about this? I'm not sure a web IDE quite fits our content requirements? I feel it falls into the bucket of "most websites use JS, but it doesn't mean they're directly related"?

indus commented:
@MattIPv4 It is borderline but I tend to accept it...
Welcome https://spark.js.org
